earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+SCRIPTDIR=`dirname $0`
+
+if [ ! -d Imagery ]; then
```

Comment:
Are you expecting to be in a specific directory when you run this script? What is `Imagery` relative to?